### ISSUE TITLE
I need help, I can't make it work on Windows!

### ISSUE THREAD

**Issue by fallsdevil** [syncthing/syncthing#9871]
### What happened?

I've been using syncthing for a long time, but I saw that it had updates (I used version 1.27.0) and it worked normally, when I started my laptop it started and a pop-up appeared saying that. but I had installed it manually, I did the normal download and ran it, but as there were updates I decided to try to uninstall it and install it using the terminal's "winget". It happens that every time I start the laptop, syncthing opens the terminal to run, and this is not the biggest problem, but the fact that I CLOSE the terminal, it interrupts the process. I've already researched, I've tried to fix it with AI, and nothing works, I had installed the stable version: "1.28.1", but I also installed previous versions and that's the same thing. in short, all the options I install I can't perform as before, even downloading the one recommended on the website, "windows-setup" works, it appears that there is an error with something: "WSH", I will leave a printscreen, and now I can't do it anymore use syncthing...

![image](https://github.com/user-attachments/assets/8b9db38a-94bf-40aa-a06f-1a1337704caf)

### Syncthing version

v1.28.1

### Platform & operating system

Windows 11, 64bits, AMD

**Comment by calmh** [comment]
This looks more like a support request than a clear bug or feature request. Please use the forum - https://forum.syncthing.net/ - for support, first by searching for any similar issues and then by posting a new one when required. The forum has a lot more eyes and friendly people on it than the issue tracker.

Thanks!

**Comment by tomasz1986** [comment]
You may also want to ask for help at https://github.com/Bill-Stewart/SyncthingWindowsSetup/issues, as this problem is about the Windows installer and has nothing to do with Syncthing proper.

**Comment by rujorgensen** [comment]
@fallsdevil did you find a solution 🙂?

**Comment by tomasz1986** [comment]
@rasa Why reopen? The issue is related specifically to the Windows installer, which relies on Windows Script Host.

**Comment by fallsdevil** [comment]
@rujorgensen yes, I had to install an old version of "Syncthing Windows Setup" and it recognized my existing settings and devices and can start by turning on Windows (:

**Comment by rasa** [comment]
> [@rasa](https://github.com/rasa) Why reopen?

@tomasz1986 Sorry, I reopened by mistake. Won't happen again.
